modules/core/src/viewports/globe-viewport.ts

```diff
@@ -215,15 +215,15 @@ export default class GlobeViewport extends Viewport {
     return [lng, lat, Z];
   }
 
-  projectFlat(xyz: number[]): [number, number] {
-    return xyz as [number, number];
+  projectFlat(xyz: [number, number, number]): [number, number, number] {
+    return xyz;
```

Comment:
Does "flat" mean 2D in this context? This 2D typing seemed intentional here.

Reply:
Fixed.